### ISSUE TITLE
Upgrade to latest version of Chrome

### ISSUE THREAD

**Issue by DinisCruz** [WhiteHatSecurity/Aviator#39]
In [You probably shouldn't be using the WhiteHat Aviator browser if you’re concerned about security and privacy.](https://plus.google.com/+JustinSchuh/posts/69qw9wZVH8z) there is a mention that ***Aviator is perennially at least two major releases behind Chrome,*** which is a really big issue.

I would say that it should be a **key design decision for the Aviator browser to have version parity with Chrome releases**, specially when there is a release (major or minor) that introduces security issues. 

In fact what should happen, is that in addition to having a [CI solution for Aviator](https://github.com/WhiteHatSecurity/Aviator/issues/38), a similar batch of **tests should be executed on every Chromium commit** (after merging them with the latest Aviator specific code)

That would provide advanced warning to what is going to break (or work) in the next Chromium release. 

In a nutshell , this is the workflow that is needed:

1. New Chromium commit
2. Pull Chromium changes into special Aviator branch
3. Apply Aviator changes on top (or merge Chromium commit into Aviator code)
4. Push changes into Aviator branch
5. Travis picks up new commit (in Aviator branch) and runs the tests
6. Analyse results (and most likely update tests)
